### ISSUE TITLE
Making input type="date" on toggling Edit Mode of Data type = Date

### ISSUE THREAD

**Issue by l-1** [structr/structr#440]
On toggling Edit Mode on a field which has "Data type = Date" Structr-app.js transforms it into an <input type="text" class="hasDatepicker">. 
It would be nice to have an input in edit mode with a type="date" in that case. Or a possibility to enter custom type.

Also, datepicker apperas only once on first focusing and after blur it doesn't appear again.

**Comment by amorgner** [comment]
@l-1 Could you please try to reproduce the problem with the current (2.0.1) release?

**Comment by l-1** [comment]
I've reproduced the problem with a disappearing datepicker after blur on a 2.0.2-SNAPSHOT build fd1d6 (201605181524), if it's okay to use a snapshot.

Anyway, for the time-being I've decided to use date input field as text since date format in a class is dd.MM.yyyy which makes it impossible to add/edit with only a datepicker, so it requires hidden inputs and javascript which transforms dates.

**Comment by vigorouscoding** [comment]
We are closing this issue as part of a general cleanup of our GitHub repository.

Since this issue was opened, we have released Structr 6.0 and 6.1, with 6.2 approaching. We have also migrated our development to GitLab:
https://gitlab.structr.com/structr/structr

Many older issues have already been resolved in recent versions. If you are still experiencing this problem with Structr 6.x, please open a new issue in our GitLab repository and include:

- The exact version you are using
- Steps to reproduce the problem
- Relevant logs or error messages

This helps us focus on current, reproducible issues on the actively maintained platform.

Thank you for your understanding and for your contribution.
